data/team.yml

```
@@ -83,15 +83,39 @@ joe_chen:
 thorsten_ball:
   name: Thorsten Ball
   pronouns: he/him
-  role: Head of Source
+  role: Engineering Manager, Source & APIs
   reports_to: eng_lead
-  manager_role_slug: source_lead
+  manager_role_slug: engineering_manager_source_apis
   location: Aschaffenburg, Bavaria, Germany 🇩🇪
   github: mrnugget
   email: thorsten@sourcegraph.com
   links: '[@thorstenball](https://twitter.com/thorstenball), [LinkedIn](https://www.linkedin.com/in/thorsten-ball-3142b652), [thorstenball.com](https://thorstenball.com)'
   description: "Thorsten started his career as a software engineer after dropping out of college where he studied philosophy. Since then he's mainly worked for startups (flinc, ioki) in the mobility space before starting at Sourcegraph. In his spare time, he's also a writer and wrote and self-published [two](https://interpreterbook.com) [books](https://compilerbook.com) about his favorite pet topics: interpreters & compilers. He lives together with his wife and daughter in a small-ish city in Germany. He met his wife, who happened to hail from the same small town in Germany, in Australia while he was working at a shipyard."
 
+thorsten_ball_source:
+  name: Thorsten Ball
+  pronouns: he/him/his
+  role: Engineering Manager, Source
+  reports_to: eng_lead
+  manager_role_slug: source_lead
+  hide_on_team_page: true
+
+thorsten_ball_apis:
+  name: Thorsten Ball
+  pronouns: he/him/his
+  role: Engineering Manager, APIs
+  reports_to: eng_lead
+  manager_role_slug: apis_lead
+  hide_on_team_page: true
+
+thorsten_ball_search_platform:
+  name: Thorsten Ball
+  pronouns: he/him/his
+  role: Engineering Manager, Search Platform (interims)
```

```suggestion
  role: Engineering Manager, Search Platform (interim)
```

I think this should be `interim`.